lib/backend/backend_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2016 Tigera, Inc. All rights reserved.
+// Copyright (c) 2017 Tigera, Inc. All rights reserved.
```

Comment:
@aensidhe Please can you use this format for copyright statements in updated files: 

`// Copyright (c) <old year>-2017 Tigera, Inc. and other authors. All rights reserved.`  

The `and other authors` part give you credit without adding a whole new `(c)` line for every author.

If you add a new file, please use your own copyright statement (if desired) and the Apache license stanza (required).